### PR TITLE
Huge metadata fixing

### PR DESCRIPTION
**Pull request by DandelionSprout**
Inspired by #334, I decided to fix and improve the metadata of a whole slew of lists in this repo.

Most affected were "ChinaList+EasyList" (now renamed "ADT ChinaList" by me, as it does not actually include EasyList), and "Fanboy's Spanish/Portuguese". However, around 20 other lists also saw their links be changed from HTTP to HTTPS, or even replaced entirely (e.g. in Liste FR's case).

I also marked "Fanboy's Swedish" and "Fanboy's Spanish/Portuguese" as obsolete, IIRC, as they had been marked as discontinued for more than 12 months; and rewrote some particularly odd-looking descriptions.

### CODE REVIEW

File: filters/ThirdParty/filter_230_FanboyEspanol/template.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 !
-@include "http://fanboy.co.nz/fanboy-espanol.txt" /exclude="../../exclusions.txt"
\ No newline at end of file
+@include "https://www.fanboy.co.nz/fanboy-espanol.txt" /exclude="../../exclusions.txt"
```
Comment (Alex-302) [inline filters/ThirdParty/filter_230_FanboyEspanol/template.txt:2]: `https://fanboy.co.nz/fanboy-espanol.txt` is correct

File: filters/ThirdParty/filter_219_ChinaListAndEasyList/metadata.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "filterId": 219,
-  "name": "ChinaList+EasyList",
-  "description": "China List and Easy List",
+  "name": "ADT ChinaList",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_219_ChinaListAndEasyList/metadata.json:3]: `ChinaList+EasyList` is official filter's name

File: filters/ThirdParty/filter_230_FanboyEspanol/template.txt (hunk 1 above)
Comment (DandelionSprout) [reply filters/ThirdParty/filter_230_FanboyEspanol/template.txt]: Noted, accepted, and changed.

File: filters/ThirdParty/filter_230_FanboyEspanol/metadata.json [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
+  "name": "(Obsolete) Fanboy's Spanish/Portuguese",
+  "description": "Fanboy's Spanish/Portuguese",
   "timeAdded": 1404115015843,
-  "homepage": "https://adblockplus.org/subscriptions",
+  "homepage": "https://www.fanboy.co.nz/",
   "expires": "4 days",
   "displayNumber": 2,
   "groupId": 7,
-  "subscriptionUrl": "http://fanboy.co.nz/fanboy-espanol.txt",
+  "subscriptionUrl": "https://www.fanboy.co.nz/fanboy-espanol.txt",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_230_FanboyEspanol/metadata.json:10]: `https://fanboy.co.nz/fanboy-espanol.txt` is correct

File: filters/ThirdParty/filter_218_EstonianList/metadata.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {
   "filterId": 218,
   "name": "Estonian List",
-  "description": "Estonian filtr",
+  "description": "Estonian filterlist",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_218_EstonianList/metadata.json:4]: `Filter for ad blocking on Estonian sites`

File: filters/ThirdParty/filter_219_ChinaListAndEasyList/metadata.json [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+  "description": "Adblocking list for PR-China",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_219_ChinaListAndEasyList/metadata.json:4]: Chinese language sites are not in P.R.C. only.

File: filters/ThirdParty/filter_223_FanboySwedish/metadata.json [hunk 6]
```diff
@@ -1,16 +1,17 @@
 {
   "filterId": 223,
-  "name": "Fanboy's Swedish",
+  "name": "(Obsolete) Fanboy's Swedish",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_223_FanboySwedish/metadata.json:3]: Please remove `Obsolete` from name and tags.

File: filters/ThirdParty/filter_230_FanboyEspanol/metadata.json [hunk 7]
```diff
@@ -1,16 +1,18 @@
 {
   "filterId": 230,
-  "name": "FanboyEspanol",
-  "description": "Fanboys Spanish/Portuguese",
+  "name": "(Obsolete) Fanboy's Spanish/Portuguese",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_230_FanboyEspanol/metadata.json:3]: Please remove Obsolete from name and tags.

**Comment by DandelionSprout** [comment]
I believe I've taken care of the 6 other remarks of yours as well now. I was just busy with a video conference, plus I had to think a bit on what the best way to handle ChinaList+EasyList was.

**Comment by ameshkov** [comment]
@tvinzz please take a look. Changes to the filter lists descriptions should be uploaded to Crowdin.

**Pull request merged**